functions

```diff
@@ -76,7 +82,7 @@ service_create_container() {
   local PASSWORD="$(service_password "$SERVICE")"
   local DATABASE_NAME="$(get_database_name "$SERVICE")"
 
-  ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_HOST_ROOT/data:/var/lib/clickhouse" -e "CLICKHOUSE_USER=$SERVICE" -e "CLICKHOUSE_PASSWORD=$PASSWORD" -e "CLICKHOUSE_DB=$DATABASE_NAME" --env-file="$SERVICE_ROOT/ENV" -d --restart always --label dokku=service --label dokku.service=clickhouse "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
+  ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_HOST_ROOT/data:/var/lib/clickhouse" -v "$SERVICE_HOST_ROOT/$PLUGIN_CONFIG_SUFFIX/config.xml:/etc/clickhouse-server/config.xml" -v "$SERVICE_HOST_ROOT/$PLUGIN_CONFIG_SUFFIX/users.xml:/etc/clickhouse-server/users.xml" -v "$SERVICE_HOST_ROOT/$PLUGIN_CONFIG_SUFFIX/config.d/:/etc/clickhouse-server/config.d/" -v "$SERVICE_HOST_ROOT/$PLUGIN_CONFIG_SUFFIX/users.d/:/etc/clickhouse-server/users.d/" -e "CLICKHOUSE_USER=$SERVICE" -e "CLICKHOUSE_PASSWORD=$PASSWORD" -e "CLICKHOUSE_DB=$DATABASE_NAME" --env-file="$SERVICE_ROOT/ENV" -d --restart always --label dokku=service --label dokku.service=clickhouse "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
```

Comment:
Shouldn't we mount the whole `/etc/clickhouse-server` directory instead?

Reply:
@josegonzalez I suppose you could, and it'd even unlock more configuration options (like dictionaries); would you mount it like this?

```
-v "$SERVICE_HOST_ROOT/$PLUGIN_CONFIG_SUFFIX:/etc/clickhouse-server/"
```

Reply:
Yeah. Are there any other files in that folder now?

Reply:
@josegonzalez Not by default, no. I'll modify the code 👍